src/types/IStorageDepositOptions.ts

```diff
@@ -0,0 +1,10 @@
+// types
+import type IDefaultChangeOptions from './IDefaultChangeOptions';
+
+interface IStorageDepositOptions extends IDefaultChangeOptions {
+  account_id?: string;
+  registration_only?: boolean;
```

Comment:
Convert `account_id` and `registration_only` to camelCase (just to be consistent with the other parameters)

Reply:
Done